tests/regression/87-once/03-unknown.c

```diff
@@ -0,0 +1,44 @@
+// PARAM: --set ana.activated[+] pthreadOnce
+#include <pthread.h>
+#include <stdio.h>
+
+int g;
+pthread_once_t once1 = PTHREAD_ONCE_INIT;
+pthread_once_t once2; // PTHREAD_ONCE_INIT is `0`
```

Comment:
What is the relevance of `PTHREAD_ONCE_INIT` being 0? Is this an extra assumption we're now making?

Reply:
PTHREAD_ONCE_INIT corresponds to a struct with all values zero-initialized, so coincides with the default value for global variables and can be omitted.
It technically is an assumption, but we (and everyone else) makes the same assumption about `PTHREAD_MUTEX_INITIALIZER` as well, so I don't think it's worth documenting.

Reply:
> Why does it matter for us at all?

It matters in that the program has defined behavior, and that's the only thing that matters.

I'm not sure if we need some opaque thing here, it seems to work without it.

Reply:
I mean, we probably will make unsound claims about `pthread_once(&once, fun); assert(once == 0);`. Currently base analysis would prove this, but I'm doubtful that this actually holds.

Reply:
I have now fixed this. I haven't renamed `ValueDomain.Mutex` though. We already use it for condition variables anyway, and I guess `pthread_once` is just as mutex-flavored as condition variables.